setup.py

```diff
@@ -3,11 +3,15 @@
 
 setup(
     name="calc",
-    version="0.3",
+    version='0.4',
     packages=["calc"],
     install_requires=[
         "pyparsing==2.2.0",
```

Comment:
Should we be more lenient with this now?
(How) is this specified elsewhere in `edx-platform`?